projects/core/src/cart/facade/cart-data.service.ts

```diff
@@ -61,6 +61,14 @@ export class CartDataService {
   }
```

Comment:
I can't comment on the line but on 59. It's actually possible to have a | in the first part of the email (before @). It will not work if the email specified contains one. The mechanism should be more fool proof.